docs/quickstart.rst

```diff
@@ -1,116 +1,81 @@
 
     client = discord.Client(intents=discord.Intents.all())
-    slash = SlashCommand(client, auto_register=True)
-
-    guild_ids = [789032594456576001] # Put your testing server ID.
+    slash = SlashCommand(client, auto_register=True) # Declares slash commands through the client.
 
     @client.event
     async def on_ready():
         print("Ready!")
 
-    @slash.slash(name="ping", guild_ids=guild_ids)
-    async def _ping(ctx):
-        await ctx.send(content=f"Pong! ({client.latency*1000}ms)")
-
     client.run("your_bot_token_here")
 
-I'll explain what is changed:
-
-- ``guild_ids = [789032594456576001]``: This is for adding your command as a guild command. Otherwise, you need to wait for an hour to wait until your command is added.
-- ``@slash.slash(name="ping"...)`` ~ ``await ctx.send(...)``: This adds new slash command. This command basically sends an API latency.
+Let's give this a run. When you run this code, you'll see... nothing but ``Ready!``.
 
-Let's do some more complicated things:
+That's completely normal, because we haven't defined any slash commands yet.
+We can do so by adding this code shown here:
 
 .. code-block:: python
 
     import discord
-    from discord_slash import SlashCommand
-    from discord_slash import SlashCommandOptionType
-    from discord_slash.utils import manage_commands
+    from discord_slash import SlashCommand # Importing the newly installed library.
 
     client = discord.Client(intents=discord.Intents.all())
-    slash = SlashCommand(client, auto_register=True)
+    slash = SlashCommand(client, auto_register=True) # Declares slash commands through the client.
 
-    guild_ids = [789032594456576001] # Put your testing server ID.
+    guild_ids = [789032594456576001] # Put your server ID in this array.
 
     @client.event
     async def on_ready():
         print("Ready!")
 
     @slash.slash(name="ping", guild_ids=guild_ids)
-    async def _ping(ctx):
+    async def _ping(ctx): # Defines a new "context" (ctx) command called "ping."
         await ctx.send(content=f"Pong! ({client.latency*1000}ms)")
 
-    @slash.slash(name="echo", guild_ids=guild_ids, options=[manage_commands.create_option("string", "A random string.", SlashCommandOptionType.STRING, True)])
-    async def _echo(ctx, string):
-        await ctx.send(content=string)
-
     client.run("your_bot_token_here")
 
-Again, I'll explain what is changed:
+Let's explain some of the major code differences between the prior examples shown
+here to give a better understanding of what is going on:
 
-- ``from discord_slash.utils import manage_commands``: This imports utility module of this extension.
-- ``@slash.slash(name="echo"...)`` ~ ``await ctx.send(...)``: This adds another slash command. This command echos what you typed at ``string``.
+- ``guild_ids = [789032594456576001]``: This is for adding your command as a guild command.
+Otherwise, you need to wait for an hour to wait until your command is added. This is due
```

Comment:
```py
C:\Users\sions\PycharmProjects\discord-py-slash-command\docs\quickstart.rst:68: WARNING: Bullet list ends without a blank line; unexpected unindent.
C:\Users\sions\PycharmProjects\discord-py-slash-command\docs\quickstart.rst:73: WARNING: Bullet list ends without a blank line; unexpected unindent.
```
Warning happens while running docs build.

Reply:
Apologies, I've never really tinkered with the design of docs before, this is my first attempt. Fairly unsure how to fix this.

Reply:
Adding one blank line between line 67-68 and line 72-73 should work.

Reply:
Line spaces have been added.